README.md

```diff
@@ -11,117 +11,47 @@ developed.](https://img.shields.io/badge/repo%20status-Active-green.svg)](https:
 [![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=cosmos_gaia&metric=ncloc)](https://sonarcloud.io/summary/new_code?id=cosmos_gaia)
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=cosmos_gaia&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=cosmos_gaia)
 [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=cosmos_gaia&metric=coverage)](https://sonarcloud.io/summary/new_code?id=cosmos_gaia)
-[![Discord](https://badgen.net/badge/icon/discord?icon=discord&label)](https://discord.gg/cosmosnetwork)
-[![Twitter](https://badgen.net/badge/icon/twitter?icon=twitter&label)](https://twitter.com/cosmoshub)
+[![Twitter](https://badgen.net/badge/icon/Twitter?icon=twitter&label)](https://x.com/cosmoshub)
```

Comment:
If we're changing the URL to X 😢 , then maybe we can change the icon as well.

Reply:
I cannot find an icon